README.md

```diff
@@ -7,78 +7,47 @@
-training, and evaluation (see our [overview](docs/lm/overview.md) or XXX tootsie retrospective).
+Marin's primary use case is training language model like Llama, DeepSeek, Qwen, etc.
+Notably, this includes data curation, transformation, filtering, tokenization, training, and evaluation.
 
-We used Marin to train the Caravel 8B model, the first open-source 8B-class model to outperform Llama 3.1 8B.
-You can see the training script [here](https://github.com/marin-community/marin/blob/main/experiments/tootsie/exp600_tootsie.py) or read the [retrospective](docs/lm/tootsie-retrospective.md) XXX.
-
-Marin was originally created by Stanford's [Center for Research on Foundation Models](https://crfm.stanford.edu/) (CRFM),
-but you can [join us](#get-involved)!
+We used Marin to train the first open-source 8B parameter model to outperform Llama 3.1 8B.
+You can see the training script [here](https://github.com/marin-community/marin/blob/main/experiments/tootsie/exp600_tootsie.py) or read the [retrospective](docs/reports/tootsie-retrospective.md).
 
 <!--marin-intro-end-->
 
-## Documentation
-
 The documentation for Marin is available on [ReadTheDocs](https://marin.readthedocs.io/en/latest/) or in the [`docs/`](docs/) folder.
 
 <!--marin-first-steps-start-->
 
-To learn more about Marin and how it works, you can:
+To get started with Marin:
 
+- [Install](docs/tutorials/installation.md) Marin.
 - Train a [tiny language model](docs/tutorials/first-experiment.md) using Marin.
-- See how to run a [DCLM 1B/1x](docs/tutorials/train-an-lm.md) experiment using Marin.
-- Learn how to join Marin's [Speed Run](docs/tutorials/submitting-speedrun.md) to try to find fast, scalable ways to train language models.
-- Read about Marin's key concepts and principles in [Concepts](docs/explanation/concepts.md)
-- Follow the [Executor 101](docs/tutorials/executor-101) tutorial to get a feel for how Marin works.
-- Learn about the [Executor framework](docs/explanation/executor.md): how to manage Python libraries, run big parallel jobs using Ray, how versioning works, etc.
-- Read about [Experiments](docs/explanation/experiments.md): how we use the executor framework to run machine learning experiments.
+- See how to run a much larger [DCLM 1B/1x](docs/tutorials/train-an-lm.md) experiment using Marin.
+- See a [summary of the experiments](docs/reports/index.md) we've run.
+- Participate in the Marin [Speedrun competition](docs/tutorials/submitting-speedrun.md) to try to find the most efficient way to train a language model.
+- Try out the [Marin Datashop](docs/tutorials/datashop.md) to contribute and create data for your use case.
+- Join the [Marin Discord](https://discord.gg/J9CTk7pqcM) to chat with the community.
 
 <!--marin-first-steps-end-->
 
-## Getting Started
-
-Full documentation on setup is available on [ReadTheDocs](https://marin.readthedocs.io/en/latest/tutorials/getting-started/).
-
-### Installation
```

Comment:
Let's just redirect people to the installation page.  I don't think we can have a simple set of instructions that works for everyone (for example, these don't work for me), unless we make an installation script with a bunch of if statements.

Reply:
ok sure